test/pageobjects/HomePage.js

```diff
@@ -2,38 +2,72 @@ import Base from './Base'
 
 class HomePage extends Base {
     get ToolsQaImg() { return $('img') }
-    get h5() { return $('h5')}
-    get ElementsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(1)') }
-    get FormsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(2)') }
-    get AFWEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(3)') } 
-    get WidgetsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(4)') }
-    get InteractionsEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(5)') }
-    get BookStoreAppEntity() { return $('#app > div > div > div.home-body > div > div:nth-child(6)') }
-    get MainHeader() { return  $('.main-header')}
+
+    get h5_Elements() { return $('h5=Elements') }
+    get h5_Forms() { return $('h5=Forms') }
+    get h5_AFW() { return $('h5=Alerts, Frame & Windows') }
+    get h5_Widgets() { return $('h5=Widgets') }
+    get h5_Interactions() { return $('h5=Interactions')}
+    get h5_BookStoreApp() { return $('h5=Book Store Application')}
+
+    get MainHeader1() { return $('.main-header=Elements')}
+    get MainHeader2() { return $('.main-header=Forms')}
+    get MainHeader3() { return $('.main-header=Alerts, Frame & Windows')}
+    get MainHeader4() { return $('.main-header=Widgets')}
+    get MainHeader5() { return $('.main-header=Interactions')}
+    get MainHeader6() { return $('.main-header=Book Store')}
```

Comment:
Are you able to use this elements with this kind of selectors? An why do you have duplicates of the header elements?

Reply:
Yes, I am able to use this elements with this kind of selector. I have duplicates of header elements because all the six entities on homepage have same selector.